planner/logical.go

```diff
@@ -134,6 +146,10 @@ func createLogicalPlan(spec *flux.Spec, a Administration) (*PlanSpec, error) {
 		return nil, err
 	}
 
+	if err := v.plan.BottomUpWalk(ComputeBounds); err != nil {
```

Comment:
I would have expected this to be in the physical planner, computed just before returning the final plan.  The reason is, there may be rules we apply that refine the time bounds on the plan nodes, for example, if we can push a `range` down through a `window`.

Reply:
Yes you're right, I'll move this to end of planning.